CHANGES.rst

```diff
@@ -176,6 +176,10 @@ API Changes
     - Removed ``startColumn`` and ``endColumn`` arguments to the ``FITS_record``
       constructor.
 
+  - FITS writers now accept an ``overwrite`` argument. If ``clobber``
+    is used, an ``AstropyDeprecationWarning`` is issued. If both are
+    used, a ``TypeError`` is raised. [#5171]
```

Comment:
I would rephrase this to say that 'The ``clobber`` argument in FITS writers has been renamed to ``overwrite``' (then leave the bit about deprecation).